src/v/cluster/rm_stm.cc

```diff
@@ -597,6 +597,11 @@ rm_stm::replicate(
     }
 }
 
+ss::future<> rm_stm::stop() {
```

Comment:
can this patch be squashed into `kafka/tx: integrate pre-prepare auto aborts with rm_stm`?